pkg/server/router/issuance.go

```diff
@@ -28,17 +31,35 @@ func NewIssuanceRouter(svc framework.Service) (*IssuanceRouter, error) {
 // @Accept       json
 // @Produce      json
 // @Param        id   path      string  true  "ID"
-// @Success      200  {object}  issuing.GetIssuanceTemplateResponse
+// @Success      200  {object}  issuing.IssuanceTemplate
 // @Failure      400  {string}  string  "Bad request"
 // @Router       /v1/issuancetemplates/{id} [get]
-func (ir IssuanceRouter) GetIssuanceTemplate(ctx context.Context, w http.ResponseWriter, req *http.Request) error {
-	return nil
+func (ir IssuanceRouter) GetIssuanceTemplate(ctx context.Context, w http.ResponseWriter, _ *http.Request) error {
+	id := framework.GetParam(ctx, IDParam)
+	if id == nil {
+		return framework.NewRequestError(
+			util.LoggingNewError("cannot get issuance template without an ID"), http.StatusBadRequest)
+	}
+
+	issuanceTemplate, err := ir.service.GetIssuanceTemplate(&issuing.GetIssuanceTemplateRequest{ID: *id})
+	if err != nil {
+		return framework.NewRequestError(
+			util.LoggingErrorMsg(err, "getting issuance template"), http.StatusInternalServerError)
+	}
+	return framework.Respond(ctx, w, issuanceTemplate.IssuanceTemplate, http.StatusOK)
 }
 
 type CreateIssuanceTemplateRequest struct {
 	issuing.IssuanceTemplate
 }
 
+func (r CreateIssuanceTemplateRequest) ToServiceRequest() *issuing.CreateIssuanceTemplateRequest {
+	return &issuing.CreateIssuanceTemplateRequest{
+		ID:               uuid.NewString(),
```

Comment:
In our other objects (like schema) we don't have the UUID in the request and we create it on more layer down in the CreateSchema function which is closer to DB.

Shouldn't matter either way, but eventally we should be consistent